lib/ws-kernel-picker.coffee

```diff
@@ -42,17 +42,21 @@ class WSKernelPicker
         @_onChosen = onChosen
 
     toggle: (grammar) ->
+        gateways = Config.getJson('gateways', [])
+        if _.isEmpty gateways
+            atom.notifications.addError 'No remote kernel gateways available',
+                dismissable: true
```

Comment:
I think `dismissable: false` should be the way to go. Then the notification gets dismissed after a few second automatically. This is the default behavior so removing this line will work.